tools/encyclopedia/encyclopedia_prosit_csv_to_library.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="encyclopedia_prosit_csv_to_library" name="ConvertPrositCSVToLibrary" version="@VERSION@.0" python_template_version="3.5">
+    <description>Convert Prosit/Spectronaut CSV to EncyclopeDIA DLIB library</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        EncyclopeDIA -convert -prositcsvtolibrary
+        -i '$i'
+        -f '$f'
+	-o '$output'
+        #if $istest == 'yes'
+        && sqlite3 -readonly '$output' 'select distinct ProteinAccession from peptidetoprotein;' > '$test_output'
+        #end if
+    ]]></command>
+    <inputs>
+        <param argument="-i" type="data" format="csv" label="Prosit output csv file"/>
```

Comment:
Prosit is an external tool and we can not use a TSV here?

Reply:
Not yet any way. Prosit requires CSV.